public/docs/_examples/component-styles/dart/lib/hero.dart

```diff
@@ -0,0 +1,6 @@
+class Hero {
+  String name;
+  List<String> team;
+  bool active;
+  Hero(this.name, this.team) {}
```

Comment:
Omit the body `{}` and replace it by ';' when you don't have a constructor body.